src/main/java/com/google/json/JsonSanitizer.java

```diff
@@ -541,16 +541,41 @@ private void sanitizeString(int start, int end) {
             }
           }
           break;
-        // Embedding.  Disallow </script and ]]> in string literals so that
-        // the output can be embedded in HTML script elements and in XML CDATA
-        // sections.
-        case '/':
-          // Don't over escape.  Many JSON bodies contain innocuous HTML
-          // that can be safely embedded.
-          if (i > start && i + 2 < end && '<' == jsonish.charAt(i - 1)
-              && 's' == (jsonish.charAt(i + 1) | 32)
-              && 'c' == (jsonish.charAt(i + 2) | 32)) {
-            insert(i, '\\');
+        // Embedding. Disallow <script, </script, <!--, --> and ]]> in string
+        // literals so that the output can be embedded in HTML script elements
+        // and in XML CDATA sections without affecting the parser state.
+        // References:
+        // https://www.w3.org/TR/html53/semantics-scripting.html#restrictions-for-contents-of-script-elements
+        // https://www.w3.org/TR/html53/syntax.html#script-data-escaped-state
+        // https://www.w3.org/TR/html53/syntax.html#script-data-double-escaped-state
+        // https://www.w3.org/TR/xml/#sec-cdata-sect
```

Comment:
Thanks.  This is a better set of URLs than those at

https://github.com/OWASP/java-html-sanitizer/blob/3d2fc674d72de48648a8787934c1df33ffad2669/src/main/java/org/owasp/html/HtmlStreamRenderer.java#L291-L292

so I'll update those separately.